development/process.md

```diff
@@ -102,12 +101,21 @@ keep the amount of rework from the contributor to a minimum.
       website are updated.
     * Finally the release is announced.
 
+## Code style
+
+All contributions must follow the project's code style guidelines to ensure
+consistency and readability. For Java code, follow the [Trino Java code style guide](https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#code-style).
+For SQL and documentation, refer to the respective sections in the [development guide](https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md).
+
+Before submitting a pull request, please format your code and check for style issues.  
```

Comment:
Not sure we need this here vs in the process higher up or at all ..

Reply:
If we leave .. we should reword to avoid "please" and be direct .. if the code it not formatted it will actually not even pass the build .. you could just say that its a requirement for a PR therefore